### ISSUE TITLE
Windows Machine-Wide MSI Installer should default to using the ALLUSERS=1 Property

### ISSUE THREAD

**Issue by asjimene** [desktop/desktop#10082]
### Describe the feature or problem you’d like to solve

The Desktop Installer MSI should install with the ALLUSERS property set to "1". In it's current state, the MSI by default only installs for the user running the application, even though the files are installed System wide. This means that another user will effectively be unable to uninstall the application.

### Proposed solution

Set the ALLUSERS property in the MSI to 1 by default. This will direct the MSI to register to the machine instead of the user installing the application.

### Additional context

Consider the following situation:
A systems admin manually installs the GitHub Desktop Machine-Wide Installer MSI using their administrator account (using the command `msiexec /i GitHubDesktop.msi /qn`
Later, an updated GitHub Desktop Machine-Wide installer MSI is pushed through Active Directory or Microsoft Configuration Manager (this would use the command: `msiexec /i GitHubDesktop.msi /qn ALLUSERS=1`)

What should happen is the new MSI install should cleanly upgrade the existing MSI installation, however, because the original installation did not use the ALLUSERS=1 flag, there will be 2 entries for the GitHub Desktop Machine Wide installer.
Seeing as the MSI is actually a "Machine-Wide Install", it seems that setting the ALLUSERS flag to 1 would be appropriate for this package.

![image](https://user-images.githubusercontent.com/3790176/85637659-a15ef480-b638-11ea-8ff7-175b223e4db0.png)

**Comment by tierninho** [comment]
@asjimene Thanks for the suggestion on this! We will look into and make any necessary adjustments.

**Comment by JordanTheITGuy** [comment]
This seems to still be an issue for properly updating devices at scale. :(

This could potentially trigger false positives in AV scans as well. 

Here is a further example where the software is installed in three different ways.

![image](https://user-images.githubusercontent.com/32438363/113770674-6a9e2a80-96f0-11eb-9865-1a219f9b0ee4.png)

**Comment by tromlet** [comment]
Yeah - we're running on a 2008 R2 Terminal Server (I know - we're working on it) and want some of our less tech-savvy users to be able to get in on some Git goodness by using Markdown for documentation and procedures, while being able to track changes. But I'm seeing GitHub Desktop consistently installing to... the `C:\Users\$user\AppData\Local\GitHubDesktop` directory instead of the `C:\Program Files\GitHub Desktop` directory. This makes it a little bit unwieldy for me, the sysadmin, to make sure everyone's up-to-date and on the same version.

Also that the .msi file installes the "machine-wide" .exe installer to the `C:\Program Files (x86)\GitHub Desktop Deployment` directory (without communicating... anything to the user...) instead of just... installing the software machine-wide as per expected .msi fashion is more than a little confusing - and there isn't a whole lot of documentation on this. :/

EDIT: We will ultimately be moving to a more recent version of Windows Server with RDS, so getting this to work optimally within the context of Remote Desktop Services would be... awesome. It's a super convenient way to run a business.

**Comment by rwidmark** [comment]
Yeah, this is a issue I don't understand why you can't install GitHub Desktop under Programfiles instead of AppData.
It don't make sens and make a lot of troubles for enterprises that don't allow suers to have local admin on the Windows machine

**Comment by rwidmark** [comment]
Any news on this?

**Comment by tromlet** [comment]
> Any news on this?

lol yeah, we have a Server 2019 Standard x64 box now for our Terminal Server, aaaand... doesn't... look like we're able to use GitHub Desktop still. >:/

**Comment by somehume** [comment]
Any change? I need to install GitHub Desktop on a bunch of lab machines.
